### PR TITLE
feat: #1252 update statistics in hours

### PR DESCRIPTION
**Pull request by AKiroi**
image

### CODE REVIEW

File: mw-webapp/src/logic/wayPage/wayStatistics/statisticBlock/statisticItem/StatisticItem.tsx [hunk 1]
```diff
@@ -60,11 +63,14 @@ interface StatisticItemProps {
  * StatisticItem
  */
 export const StatisticItem = (props: StatisticItemProps) => {
+  const hours = props.statisticItem.value / MINUTES_IN_HOUR;
+  const formattedHours = props.statisticItem.value > MINUTES_THRESHOLD ? Math.round(hours) : hours.toFixed(DECIMAL_PLACES);
```
Comment (Ekaterina1994) [inline mw-webapp/src/logic/wayPage/wayStatistics/statisticBlock/statisticItem/StatisticItem.tsx:67]: Need create util function

File: mw-webapp/src/logic/wayPage/wayStatistics/statisticBlock/statisticItem/StatisticItem.tsx [hunk 2]
```diff
@@ -60,11 +63,14 @@ interface StatisticItemProps {
...
+
   return (
     <div className={clsx(styles.statisticItem, styles[props.type ?? StatisticItemType.PRIMARY])}>
       <span className={styles.statisticValue}>
         {props.convertToHours
```
Comment (Ekaterina1994) [inline mw-webapp/src/logic/wayPage/wayStatistics/statisticBlock/statisticItem/StatisticItem.tsx:72]: Need convert to hours also other  statistics (писала в телеграме)

**Review by Ekaterina1994 — CHANGES_REQUESTED** [review]

File: mw-webapp/src/logic/wayPage/wayStatistics/statisticBlock/statisticItem/StatisticItem.tsx (hunk 1 above)
Comment (AKiroi) [reply mw-webapp/src/logic/wayPage/wayStatistics/statisticBlock/statisticItem/StatisticItem.tsx]: done

File: mw-webapp/src/logic/wayPage/wayStatistics/statisticBlock/statisticItem/StatisticItem.tsx (hunk 2 above)
Comment (AKiroi) [reply mw-webapp/src/logic/wayPage/wayStatistics/statisticBlock/statisticItem/StatisticItem.tsx]: done

File: mw-webapp/src/logic/wayPage/wayStatistics/statisticBlock/statisticItem/StatisticItem.tsx [hunk 3]
```diff
@@ -63,8 +63,8 @@ export const StatisticItem = (props: StatisticItemProps) => {
   return (
     <div className={clsx(styles.statisticItem, styles[props.type ?? StatisticItemType.PRIMARY])}>
       <span className={styles.statisticValue}>
-        {props.convertToHours
-          ? `${Math.round(props.statisticItem.value / MINUTES_IN_HOUR)}h`
+        {props.type === StatisticItemType.SECONDARY || props.convertToHours
```
Comment (Ekaterina1994) [inline mw-webapp/src/logic/wayPage/wayStatistics/statisticBlock/statisticItem/StatisticItem.tsx:66]: Let's move to const this difficult logic)

Comment (AKiroi) [reply mw-webapp/src/logic/wayPage/wayStatistics/statisticBlock/statisticItem/StatisticItem.tsx]: done

**Review by Ekaterina1994 — APPROVED** [review]

**Review by tritonJS826 — APPROVED** [review]

**Pull request merged**